docs/infrastructure/README.md

```diff
@@ -62,6 +62,10 @@ repositories is an [ongoing project][legacy-repository-migration].
 For our email (`@studentrobotics.org`) accounts as well as document sharing we
 use Google Workspace. Google Groups are used for various mailing lists.
 
+**Note:** As of early 2022, the Infrastructure Team do not have administrative
```

Comment:
This implies it will be the state going forward rather than a temporary limitation.
A potential better wording would be.

**Note:** The Infrastructure Team do not yet have administrative access to Google Suite, so account management requests must be made to the trustees instead. The Infrastructure team will be taking this over at some point in the future.

Reply:
Yup 👍. I was struggling with a better wording, your suggestion is good.

Reply:
54ec131